shell/platform/darwin/ios/framework/Headers/FlutterEngineGroup.h

```diff
@@ -8,6 +8,36 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/** Options that control how a FlutterEngine should be created. */
+FLUTTER_DARWIN_EXPORT
+@interface FlutterEngineGroupOptions : NSObject
+
+/**
+ * The name of a top-level function from a Dart library.  If this is FlutterDefaultDartEntrypoint
+ * (or nil); this will default to `main()`.  If it is not the app's main() function, that function
+ * must be decorated with `@pragma(vm:entry-point)` to ensure themethod is not tree-shaken by the
+ * Dart compiler.
+ */
+@property(nonatomic, copy) NSString* entrypoint;
```

Comment:
These properties are nullable right?  You should specify that `@property(nonatomic, copy, nullable)`

Reply:
Done